src/main/resources/META-INF/plugin.xml

```diff
@@ -29,6 +29,7 @@
        on how to target different products -->
   <depends>com.intellij.modules.lang</depends>
   <depends>org.jetbrains.plugins.terminal</depends>
+  <depends>org.jetbrains.plugins.yaml</depends>
```

Comment:
I would have expected that line 38 would be updated as well, no ?

Reply:
No, i tried changing it to yaml but it makes the completion stop working. At first i thought that that was the only option to be changed to make it works with yaml file but no... i really don't know how it works or maybe yaml language has some issue